src/components/nve-link-card/nve-link-card.component.ts

```diff
@@ -0,0 +1,146 @@
+import styles from './nve-link-card.styles';
+
+/**
+ * Komponent som brukes til å vise lenkekort. Brukes til å navigere til interne, eksterne sider, laste ned filer, eller sende e-post.
+ *
+ * Egenskaper som `clickAction` bestemmer oppførselen når kortet klikkes, mens `title` og `additionalText` brukes til å vise innholdet på kortet.
+ */
+@customElement('nve-link-card')
+export default class NveLinkCard extends LitElement implements INveComponent {
+
+  /** Tittel som vises øverst på kortet */
+  @property({ reflect: true }) title: string = "";
+
+  /** Tilleggsbeskrivelse som vises under tittelen */
+  @property({ reflect: true }) additionalText?: string;
+
+  /** Størrelse på kortet, kan være 'small', 'medium' eller 'large' */
+  @property({ reflect: true }) size: 'small' | 'medium' | 'large' = 'medium';
+
+  /** Variant som bestemmer stilen på kortet: 'primary', 'contrast', eller 'secondary' */
+  @property({ reflect: true }) variant: 'primary' | 'contrast' | 'secondary' = 'primary';
+
+  /** Test ID som kan brukes i testing og sporing */
+  @property({ reflect: true }) testId: string | undefined = undefined;
+
+  /** Definerer hva som skjer når kortet klikkes: 'internal' (intern lenke), 'download' (nedlasting), 'external' (ekstern lenke), eller 'mail' (e-post) */
+  @property({ reflect: true }) clickAction: 'internal' | 'download' | 'external' | 'mail' = 'internal';
+
+  /** Lenken som brukes for handlinger som intern/ekstern navigering eller e-post */
+  @property({ reflect: true }) href: string | undefined = undefined;
+
+  /** Valgfri nedlastingsfunksjon som kan overstyres for å implementere spesifikk nedlastingslogikk */
+  @property() downloadHandler: () => void = this.defaultDownloadHandler;
+
+  static styles = [styles];
+
+  constructor() {
+    super();
+  }
+
+  /**
+   * Standard nedlastingsfunksjon som brukes hvis `downloadHandler` ikke er overstyrt. 
+   * Bruker lenken (`href`) til å laste ned en fil.
+   */
+  private async defaultDownloadHandler() {
+    if (this.href) {
+      const fileUrl = this.href;
+      const fileName = fileUrl.split('/').pop() || 'download';
+
+      try {
+        const response = await fetch(fileUrl, {
+          mode: 'cors',
+        });
+
+        if (!response.ok) {
+          throw new Error(`Failed to fetch file: ${response.statusText}`);
+        }
+
+        const blob = await response.blob();
+        const url = URL.createObjectURL(blob);
+
+        const a = document.createElement('a');
+        a.href = url;
+        a.download = fileName;
+
+        document.body.appendChild(a);
+        a.click();
+
+        document.body.removeChild(a);
+        URL.revokeObjectURL(url);
+
+      } catch (error) {
+        console.error('Download failed:', error);
+      }
+    }
+  }
+
+  /**
+   * Denne funksjonen bestemmer hva som skjer når kortet blir klikket, basert på verdien av `clickAction`.
+   */
+  private handleClick() {
+    switch (this.clickAction) {
```

Comment:
Denne bør ta imot eventet, og så sjekke om ctrl, shift, alt eller meta er aktive. Da bør den ikke navigere

Reply:
Bør også ha støtte for en custom click-handler, hvis ikke så kan den ikke brukes i Vue , siden navigasjon der bør skje via vue-router (dersom vi bare navigerer via href så lastes vue-appen på nytt

Reply:
Istællet før att stoppa navigera så har jag lagt til så att den føljer standard beteende: 
   * Ctrl/Cmd + Klikk: Åpner lenken i en ny fane.
   * Shift + Klikk: Åpner lenken i et nytt vindu.
   * Alt + Klikk: Laster ned lenken (hvis det er en fil).